cypress.json

```diff
@@ -1,7 +1,3 @@
 {
-  "env": {
-    "apiURL": "localhost:3000/.netlify/functions/server"
-  },
-  "baseUrl": "http://localhost:8000",
-  "chromeWebSecurity": false
+
```

Comment:
do you still need this file at all?

Reply:
I wanted to keep it around just in case we needed a global config, but yeah it's possible we don't even need it now

Reply:
nope it looks like removing this file breaks things